src/Microsoft.ML.Data/Transforms/Normalizer.cs

```diff
@@ -352,7 +352,7 @@ internal static void SaveType(ModelSaveContext ctx, ColumnType type)
                 ctx.Writer.Write(vectorType?.Size ?? 0);
 
                 ColumnType itemType = vectorType?.ItemType ?? type;
-                var itemKind = itemType.RawKind;
+                itemType.RawType.TryGetDataKind(out DataKind itemKind);
                 Contracts.Assert(itemKind == DataKind.R4 || itemKind == DataKind.R8);
                 ctx.Writer.Write((byte)itemKind);
             }
```

Comment:
What a very, very strange format... if there are only two choices, why not just write a bool?

Anyway, I think if we want to get rid of datakind, I think the best choice would be to write the bytes as constants... so I guess, `ctx.Writer.Write((byte)(itemType == NumberType.Double ? 10 : 9))` or something like that. I guess we'd need a similar test during the deserialization. Not sure if you're up for this in this PR...